### DISCUSSION TITLE
New Radial Menu Options

### DISCUSSION THREAD

**Discussion by Eurymachus** [gameserverapp/Platform#641] [Suggestions]
The new radial menu options are great. However as discussed in the discord due to WC's poor implementation of this feature the radial option overrides the intended behaviour for quick switching ammo when a weapon is equipped.

I propose two non-mutually exclusive options;
1. Config file option for EnableGSARadialMenu=true/false
2. Move the radial options into their own craftable tool/weapon, like many other mods. This way the radial wheel won't interfere with the current user experience.

Thanks for all your amazing work ,o7

**Comment by 3mptylord** [comment]
+1

I'd also appreciate either the option to disable specific options and/or for the radial-menu to respect the Integration App options.

For example: I have the Feedback and Voting featured disabled, but the radial menu is now allowing players to access them.

**Comment by mvaessen** [comment]
we are planning to make it configurable via the integration mod settings page, where you can enable/disable specific buttons

**Comment by mvaessen** [comment]
![Screenshot 2023-12-13 at 17 20 56](https://github.com/gameserverapp/Platform/assets/13402423/dbc91bcb-8250-4160-938a-564eee826327)

this feature will be deployed to GSA and whenever the mod update is accepted it will listen to your settings.

**Reply by mvaessen** [reply]
This is now live

**Reply by Eurymachus** [reply]
If we disable all the options does this disable the radial wheel entirely?

**Reply by mvaessen** [reply]
yes, that's what it said on the screenshot ;)
